docs/General/Manuals/client-manual.rst

```diff
@@ -851,10 +851,163 @@ sure to have a close look at the in game help on Terrain for more information.
 Multiplayer Menu
 ^^^^^^^^^^^^^^^^
 
+The :guilabel:`Multiplayer` menu has a collection of functions to aid certain multi-player games. Many of
+the options are specifically tailored to the MP2c and WarCiv rulesets.
+
+The :guilabel:`Multiplayer` menu has the following options:
+
+Delayed GoTo
+    Give a unit orders to move at a specific time in the turn. This assumes that the turn is time based.
+
+Delayed Orders Execute
+    Execute an action by a unit at a specific time in the turn. This assumes that the turn is time based.
+
+Clear Orders
+    Clear any delayed orders from the above two menu items.
+
+Add All Cities to Trade Planning
+    All all current cities in your into an advanced trade planning array. This is used by the WarCiv
+    ruleset.
+
+Calculate Trade Planning
+    Run a trade effectiveness algorithm across all of the cities in the trade plan to determine the best
+    routes. This is used by the WarCiv ruleset.
+
+Add/Remove City
+    Add or remove a city from the trade planning array.
+
+Clear Trade Planning
+    Clear all trade planning to start over.
+
+Automatic Caravan
+    Any :unit:`Caravan` units built by a city will follow the trade planning output for that city. This is
+    used by the WarCiv ruleset.
+
+Set/Unset Rally Point
+    Set or remove a rally point to easily send units, once produced, to a specific spot on the game map.
+
+Quick Airlift
+    Depending on what is selected in the menu below, you can quickly airlift a unit to a destination city.
+
+Unit Type for Quickairlifting
+    Select the type of unit that will be quick airlifted by the menu above.
+
+Default Action vs Unit
+    A sub-menu will show some optional actions that a unit should do by default against another unit.
+    The default is :strong:`Ask`.
+
+Default Action vs City
+    A sub-menu will show some optional actions that a unit should do by default against a city.
+    The default is :strong:`Ask`.
 
 Civilization Menu
 ^^^^^^^^^^^^^^^^^
 
+The :guilabel:`Civilization` menu is used to gain access to many fuctions of your empire. You can load up
+pages for units, cities, nations, etc; change the form of government and see how your are doing compared to
+your opponents with the demographics report.
+
+The :guilabel:`Civilization` menu has the following options:
+
+Tax Rates
+    Selecting this menu item will bring up a dialog box allowing you to set the rate in percentage points for
```

Comment:
When I was playing for the first time, I was confused by this, I thought, more tax on science, means lesser science. There is a distinction between tax on something and how the government is splitting the tax money, fc21 means the latter and that should probably be made clearer. 

Side node: Maybe the problem is with the name "tax rates", since it is more like "tax budget".

Reply:
I added a note. I used the "national budget" instead of tax